style.css

```diff
@@ -62,9 +57,67 @@ main {
   margin-top: 1rem;
 }
 
-.imgLogo {
-  margin-right: 1rem;
-  margin-top: 1rem;
+.bar {
+  display: block;
+  width: 20px;
+  margin: 3px auto;
+  height: 5px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+  background-color: #6070ff;
+}
+
+@media screen and (min-width: 330px) and (max-width: 599px) {
+  .hamburger {
+    display: block;
+    cursor: pointer;
+    position: fixed;
+    top: 1rem;
+    right: 1rem;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: #fff;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: #fff;
+  }
+
+  .nav-menu {
+    position: fixed;
+    right: -100%;
+    top: -1px;
+    flex-direction: column;
+    width: 100%;
+    background-color: #6070ff;
+    text-align: left;
+    transition: 0.3s;
+    height: 100vh;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(8px);
+  }
+
+  .nav-item {
+    margin-top: 2rem;
+    padding-left: 1rem;
+    width: 351px;
+  }
+
+  .nav-menu.active {
+    right: 0;
+  }
+
+  .nav-link {
+    color: #fff;
+    font-size: 1.7rem;
+  }
```

Comment:
- [ ] Amazing job with the scrolling effects. There are some small details remaining to match the Figma template. Kindly update the `font-weight` on the nav links and give a bit more space between the close button and the nav links. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/215809244-4826ee40-fec8-40e5-b643-aa1d1893cb05.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/215809265-18d92583-f06c-439d-ab48-a9ae479fd40e.png" width="300">|

Reply:
thank you for the suggestion